tf2x.yml

```diff
@@ -0,0 +1,15 @@
+name: tf2.10X
+channels:
+  - defaults
+  - conda-forge
+  - anaconda
+dependencies:
+  - python=3.9
```

Comment:
Wasn't our goal of redesigning to omit `=`?

Reply:
when dealing with complex dependencies, it is sometimes difficult to omit every =.

Reply:
I thought it was difficult to retain it.

Reply:
The issue is there is particular versions of h5py and hdf5 that are needed to load the models, and if we omit the specific python version and TF version it will install a later version of python which depends on newer versions of those packages. This ended up causing issues, so it was easier to specify the version of a couple of things', so that we could use the versions of those packages we need.